### PR TITLE
Sync triangle with problem-specifications

### PR DESCRIPTION
**Pull request by kytrinyx**
The first commit regenerates the test suite to minimize changes when doing the sync.

The sync brings in improvements to the instructions, ensures that
the source url in the metadata is https rather than http, and
adds a couple of tests.

### CODE REVIEW

**Review by devkabiir — APPROVED** [review]
Whatever it is your test generator is having, I wanna have some of that too! 🍻

**Comment by Stargator** [comment]
@kytrinyx any idea when we'll get a chance to look at your generator? I think @devkabiir  can't wait until he can see it for himself.

File: exercises/practice/triangle/test/triangle_test.dart [hunk 1]
```diff
@@ -4,106 +4,118 @@ import 'package:triangle/triangle.dart';
...
-
-      test('sides may be floats', () {
-        final bool result = triangle.scalene(0.5, 0.4, 0.6);
-        expect(result, equals(true));
-      }, skip: true);
-    });
-  });
+  group("Triangle: equilateral triangle - ", equilateralTriangle);
+  group("Triangle: isosceles triangle - ", isoscelesTriangle);
+  group("Triangle: scalene triangle - ", scaleneTriangle);
```
Comment (Stargator) [inline exercises/practice/triangle/test/triangle_test.dart:9]: Our preference is to use single quotes when possible

**Review by Stargator — CHANGES_REQUESTED** [review]

File: exercises/practice/triangle/test/triangle_test.dart (hunk 1 above)
Comment (kytrinyx) [reply exercises/practice/triangle/test/triangle_test.dart:9]: Oh, yes! Something went awry here.  Let me see what happened.

Comment (kytrinyx) [reply exercises/practice/triangle/test/triangle_test.dart]: I have fixed and regenerated.

**Comment by kytrinyx** [comment]
The generator is still very much an experiment. I'm constantly messing with it to figure out what it needs to do. I've used it for a handful of tracks and have locked down the basic functionality I will probably do a clean rewrite and make it publicly available. I don't think it would be useful for anyone to look at all the thrashing that is happening right now.

**Review by Stargator — APPROVED** [review]

**Pull request merged**